### ISSUE TITLE
oblivious

### ISSUE THREAD

**Issue by wrought13** [AngyNoT/AnimeSpotify#19]
**Anime** : Kara no Kyoukai Chapter 1
**Song type** : Ending
**Season** : 2007
**Title** : oblivious
**Singer** : Kalafina
**Spotify link** : https://open.spotify.com/track/1QgXBRLWh7NpcQg0Jxia39

**Comment by aniplaylist** [comment]
Thank you, but the song is already in the playlists!

**Comment by wrought13** [comment]
which playlist?  I was checking against Anime Songs (opening, ending, insert song), because it's the only easy way to see what you already have, and I still don't see it there.  Maybe I'm just missing it somehow though.  I have it sorted by artist, and it's not listed with the other Kalafina songs, at least.

**Comment by aniplaylist** [comment]
@TiredOfWaiting All Kalafina songs are region locked, so if you are not in Japan and want to see them in the playlist you must go to Spotify settings and enable the option (something like "shows unavailable songs").

**Comment by wrought13** [comment]
Using the links I had in the tickets, I can actually listen to all of those Kalafina songs in the US.  I do have that option enabled, and I can see 2 greyed out Kalafina songs.  This one is not one of them.  It's also not one of the active songs, like the others I sent you.  It just seems to be missing.

**Comment by aniplaylist** [comment]
I'm in France and I can't listen to the links you gave me, but I'm able to see them in the playlists ("anime songs" and also years playlists), here's a screenshot : 
![capture d ecran 2018-08-15 a 15 37 21](https://user-images.githubusercontent.com/40739069/44150929-592d148e-a0a1-11e8-8383-ff53538a207f.png)

I don't understand why you can't see them... Have you got any VPN (on your browser or computer)?

**Comment by wrought13** [comment]
Right.  I don't see "oblivious" in your screenshot either.  Maybe I'm just blind.  I just listened to it again along with the song you have highlighted, just in case that's what you meant, and they are definitely different songs.  I don't know what I'm missing here.  :-(

**Comment by aniplaylist** [comment]
Well, seems like I made a mistake about oblivious. I remember trying to add it and spotify told me that it was already in the playlist. I just added it back!

![capture d ecran 2018-08-15 a 15 51 30](https://user-images.githubusercontent.com/40739069/44151711-a7dffb12-a0a3-11e8-8286-865110b8c8f4.png)


I'm adding songs by the links you give me so it can't be different songs... 
For example, here's the link for Oblivious that i get when sharing from the playlist : 
https://open.spotify.com/track/1QgXBRLWh7NpcQg0Jxia39 

Which is the same as the one you gave above...

I think we are both missing something here :(
